backend/server/tests/planner/test_save_local_storage.py

```diff
@@ -0,0 +1,20 @@
+import json
+import requests
+
+
+PATH = "server/example_input/example_local_storage_data.json"
+
+with open(PATH, encoding="utf8") as f:
+    DATA = json.load(f)
+
+
+def test_saveLocalStorage_empty():
+    x = requests.post(
+        'http://127.0.0.1:8000/planner/saveLocalStorage', json=DATA["empty_year"])
+    assert x.status_code == 200
+
+
+def test_saveLocalStorage_simple():
+    x = requests.post(
+        'http://127.0.0.1:8000/planner/saveLocalStorage', json=DATA["simple_year"])
+    assert x.status_code == 200
```

Comment:
Can we also check the database?

I know that's not a black box test, so it might be better to expand this test later. Tell me what you think is better.

Reply:
I agree. I added something to it but it's not working...